src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -42,12 +43,12 @@ use super::filesystem::{FilesystemStatus, StratFilesystem};
 use super::mdv::MetadataVol;
 use super::thinids::ThinDevIdPool;
 
-pub const DATA_BLOCK_SIZE: Sectors = Sectors(2 * IEC::Ki);
-const DATA_LOWATER: DataBlocks = DataBlocks(512);
-const META_LOWATER_FALLBACK: MetaBlocks = MetaBlocks(512);
+pub const DATA_BLOCK_SIZE: Sectors = Sectors(2 * IEC::Ki); // 1 MiB
+pub const DATA_LOWATER: DataBlocks = DataBlocks(4096); // 2 GiB
+const META_LOWATER_FALLBACK: MetaBlocks = MetaBlocks(512); // 2 KiB
 
-const INITIAL_META_SIZE: MetaBlocks = MetaBlocks(4 * IEC::Ki);
-const INITIAL_DATA_SIZE: DataBlocks = DataBlocks(768);
+const INITIAL_META_SIZE: MetaBlocks = MetaBlocks(262144); // 1 GiB
```

Comment:
So the issue here is that the minimum total pool size is 1 GiB, see MIN_DEV_SIZE. The guidance from the thinpool docs is 1000:1 ratio for data:meta size. Meta extension should work, so can't we just rely on that?

Reply:
Ok.  Mike has suggested we oversize the meta until issues with a full meta can be worked out.  But I haven't seen any problems as long as there is space to expand the meta.